roles/stepup-gateway/tasks/main.yml

```diff
@@ -2,7 +2,7 @@
 
 
 - name: Put parameters.yml and samlstepupproviders(_parameters).yml
-  template: src={{ item }}.j2 dest={{ component_dir_name }}/app/config/{{ item }} mode=640 group={{ component_name }}
+  template: src={{ item }}.j2 dest={{ component_dir_name }}/app/config/{{ item }} mode=640
```

Comment:
Looks like I missed updating this line. Does it work when you change it to:
`template: src={{ item }}.j2 dest={{ component_dir_name }}/app/config/{{ item }} mode={{ mode_640 }}` group={{ component_group }}` ?